tools/build-tools/.eslintrc.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
     "extends": [
         "eslint:recommended",
         "plugin:@typescript-eslint/recommended",
+        // require.resolve("@fluidframework/eslint-config-fluid"),
```

Comment:
Is the intention to enable this in this PR?

Reply:
No; removed.